clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -148,11 +148,11 @@ private AbstractRequestResponse createResponseHeader() {
     }
 
     private AbstractRequest createConsumerMetadataRequest() {
-        return new ConsumerMetadataRequest("test-group");
+        return new GroupMetadataRequest("test-group");
     }
 
     private AbstractRequestResponse createConsumerMetadataResponse() {
-        return new ConsumerMetadataResponse((short) 1, new Node(10, "host1", 2014));
+        return new GroupMetadataResponse((short) 1, new Node(10, "host1", 2014));
```

Comment:
Why error code is 1 instead of 0? Also we probably should use Errors.OFFSET_OUT_OF_RANGE.code() for readability.